app/forms/contributor_form.rb

```diff
@@ -34,7 +34,8 @@ def form_properties(validate:)
         property :last_name
         property :full_name
         property :orcid
-        property :role_term
+        property :role
+        property :contributor_type
         property :_destroy, virtual: true, type: Dry::Types["params.nil"] | Dry::Types["params.bool"]
```

Comment:
We now have a radio button to tell us the type of contributor, so we do not need the `role_term` construct, which had prepended the type to the role previously.